src/view/services/command.manager.js

```diff
@@ -0,0 +1,12 @@
+export class CommandManager {
```

Comment:
It doesn't manage commands in any way: it doesn't add or delete them. It only filters and executes them within some external context (apply)